src/ignitus-Routes/notFound.js

```diff
@@ -1,20 +1,37 @@
-
 import React from 'react';
 import './Styles/style.scss';
 import { Link } from 'react-router-dom';
-
-export const Notfound = () => (
-  <div className="container-404">
-    <div className="content">
-      <p>404</p>
-      <p>We could not find what you were looking for.</p>
-      <p>
-		Please check your URL or return&nbsp;
-        <Link to="/">
+import Lottie from 'react-lottie'
```

Comment:
Lottie is not used anywhere, remove the import.

Reply:
It is used where I have added animation.It can't be removed.

Reply:
That's fine it is used @gittysachin , the problem is git ui to show changes is bit confusion, if you open file here you can see it -  https://github.com/Ignitus/Ignitus-client/blob/19c86a7a79d5d2cbfda1e3a695f16036502b4341/src/ignitus-Routes/notFound.js